crates/telio-nurse/src/config.rs

```diff
@@ -96,6 +99,17 @@ impl AggregatorConfig {
         Self {
             relay_events: features.enable_relay_conn_data,
             nat_traversal_events: features.enable_nat_traversal_conn_data,
+            state_duration_cap: Duration::from_secs(features.state_duration_cap),
+        }
+    }
+}
+
+impl Default for AggregatorConfig {
+    fn default() -> Self {
+        Self {
+            relay_events: Default::default(),
```

Comment:
Nit: I think you can use ..Default:default() here for all the remaining members, but then again if we had relay_events: false, we'd know that the default value for relay::events is deliberately chosen to be false and not just forgotten, but this is a total nit.

Reply:
I used `..Default::default()` at first but cargo was saying that it would always recurse so I changed it to setting them manually